src/components/styled/StyledTitle.js

```diff
@@ -1,7 +1,8 @@
 import styled from 'styled-components';
 
 export const StyledTitle = styled.h1`
-	color: palevioletred;
-	font-size: 1.5em;
+	margin: 0;
+	padding: 0;
+	color: black;
```

Comment:
I think its balck by default

Reply:
Ah of course it is. thanks for the hint.